README.md

```diff
@@ -23,26 +23,35 @@ go get -u github.com/cloudevents/conformance/cmd/cloudevents
 
 ## Usage
 
-`cloudevents` has two commands at the moment: `invoke` and `listen`. 
+`cloudevents` has three commands at the moment: `send`, `invoke` and `listen`. 
+
+
+### Send
+
+`send` will do a one-off creation of a cloudevent and send to a given target.
+
+```shell script
+cloudevents send http://localhost:8080 <TODO>
```

Comment:
Be good to replace this TODO and also show cloudevents send / listen example. Hmmm...I wonder if a pipe version would be useful for testing purposes...need to think about that one.

Reply:
ah! oops. will fix, thanks for finding that!